src/test/java/com/urlparser/entity/LinksFromUrlTest.java

```diff
@@ -0,0 +1,114 @@
+        "https://github.com/open-source",
+        "https://github.com/business",
+        "https://help.github.com/terms",
+        "https://help.github.com/privacy",
+        "https://github.com/join?source=button-home",
+        "https://github.com/join?plan=business&setup_organization=true&source=business-page",
+        "https://github.com/features/code-review",
+        "https://github.com/features/project-management",
+        "https://github.com/features/integrations",
+        "https://github.com/business",
+        "https://github.com/pricing",
+        "https://github.com/features/integrations",
+        "https://github.com/marketplace",
+        "https://github.com/open-source/stories/ariya",
+        "https://github.com/open-source/stories/freakboy3742",
+        "https://github.com/business/customers/mailchimp",
+        "https://github.com/open-source/stories/kris-nova",
+        "https://github.com/open-source/stories/yyx990803",
+        "https://github.com/business/customers/mapbox",
+        "https://github.com/open-source/stories/jessfraz",
+        "https://github.com/open-source",
+        "https://github.com/personal",
+        "https://github.com/business",
+        "https://help.github.com/terms",
+        "https://help.github.com/privacy",
+        "https://github.com/features#code-review",
+        "https://github.com/features#project-management",
+        "https://github.com/features#community-management",
+        "https://github.com/features#documentation",
+        "https://github.com/features#code-hosting",
+        "https://atom.io",
+        "http://electron.atom.io/",
+        "https://desktop.github.com/",
+        "https://developer.github.com",
+        "https://github.com/personal",
+        "https://github.com/open-source",
+        "https://github.com/business",
+        "https://education.github.com/",
+        "https://github.com/business/customers",
+        "https://partner.github.com/",
+        "https://community.github.com/",
+        "https://github.com/about",
+        "https://github.com/blog",
+        "https://github.com/about/careers",
+        "https://github.com/about/press",
+        "https://shop.github.com",
+        "https://github.com/contact",
+        "https://github.community",
+        "https://help.github.com",
+        "https://status.github.com/",
+        "https://github.com/site/terms",
+        "https://github.com/site/privacy",
+        "https://github.com/security",
+        "https://services.github.com/",
+        "https://github.com",
+        "https://github.com");
+
+    links = new ArrayList<>();
+    linksOrigin = new ArrayList<>();
+
+    for (URL url : origin) {
+      linksOrigin.add(url);
+    }
+
+    for (String string : array) {
+      try {
+        links.add(new URL(string));
+      } catch (MalformedURLException e) {
+        e.printStackTrace();
+      }
+    }
+  }
+
+  @Test
+  public void testTest() {
+    assertThat(linksOrigin, CoreMatchers.is(links));
```

Comment:
@undsett lets use `expected` and `actual` as a names for the vars to compare. Please fill those variables in scope of the actual test, not at `setUp`